src/app/sign-in/components/social/social.component.ts

```diff
@@ -15,6 +15,9 @@ export class SocialComponent implements OnInit {
   platform: PlatformInfo
   baseUri: string
   csrf: string
+  labelInstitutionalButton = $localize`:@@ngOrcid.signin.institutionalAccount:Access through your institution`
```

Comment:
These strings need to be added into the

i18n.pseudo.component.html and signin_en.properties